Gems/Atom/Feature/Common/Code/Source/PostProcessing/TaaPass.h

```diff
@@ -77,7 +77,6 @@ namespace AZ::Render
         RHI::ShaderInputNameIndex m_constantDataIndex = "m_constantData";
 
         AZStd::array<Data::Instance<RPI::PassAttachment>, 2> m_accumulationAttachments;
-        AZStd::array<Data::Instance<RPI::AttachmentImage>, 2> m_attachmentImages;
```

Comment:
Was this variable just completely unused?

Reply:
yes, in the TaaPass.cpp of the previous version, it holds the newly created attachment image but did nothing to it...